Changeable role attribute (#1124)
ReactTooltip/react-tooltip: docs/docs/options.mdx

| `arrowColor`            | CSS color                              | no       |                           | a CSS background color                                                                                                            | Change color of the tooltip arrow                                                                                                                               |
| `disableStyleInjection` | `boolean` or `'core'`                  | no       | `false`                   | `true` `false` `'core'`                                                                                                           | Whether to disable automatic style injection. Do not set dynamically. Check the [styling page](./examples/styling#disabling-reacttooltip-css) for more details  |

> | `role`                  | `'tooltip'` or `'dialog'`              | no       | `tooltip`                 | `'tooltip'` `'dialog'`                                                                                                            | Set ARIA role, either `tooltip` or `dialog` if the tooltip should contain focusable elements                                                                    |

Reviewer: ```suggestion
| `role`                  | ARIA role              | no       | `tooltip`                 | `'tooltip'` `'dialog'`                                                                                                            | Set ARIA role (`tooltip` by default). Setting to `dialog` is recommended if the tooltip contains focusable elements                                                                    |
```
